src/curtis.c

```diff
@@ -105,9 +117,13 @@ static veBool readRoutine(Device *device) {
 
     veItemOwnerSet(device->voltage, veVariantFloat(&v, batteryVoltage));
     veItemOwnerSet(device->current, veVariantFloat(&v, batteryCurrent));
+    veItemOwnerSet(device->power,
+                   veVariantSn32(&v, (sn32)batteryVoltage * batteryCurrent));
     veItemOwnerSet(device->motorRpm, veVariantUn16(&v, abs(motorRpm)));
     veItemOwnerSet(device->motorTemperature,
                    veVariantFloat(&v, motorTemperature));
+    // Motor Torque beyond the first decimal place is meaningless
+    veItemOwnerSet(device->motorTorque, veVariantFloat(&v, abs(roundf(motorTorque * 10)) / 10));
```

Comment:
I don't think we should alter the data. Leave it as is and the UI can choose how to format it
```suggestion
    veItemOwnerSet(device->motorTorque, veVariantFloat(&v, abs(motorTorque)));
```

Reply:
Went back and forth on that - it was the non-zero values when stationary that were annoying but I can easily take care of that in the GUI.

Feel free to make the change otherwise I will update and test at my next stop (in 90 mins or so)